content/guide/operator/meta.json

```diff
@@ -6,6 +6,7 @@
 		"index",
 		"light-mode",
 		"serverless",
-		"view-status"
+		"view-status",
```

Comment:
Think this should be view-info now because too many "status" concepts otherwise.